db/migrate/20170214024526_add_unique_songs_in_playlists.rb

```
@@ -0,0 +1,5 @@
+class AddUniqueSongsInPlaylists < ActiveRecord::Migration[5.0]
+  def change
+    add_index :playlist_songs, [:playlist_id, :song_id], unique: true
```

so you would let the insertion of multiple songs to fail, but do you think you may fail silently or handle the insertion smoothly?